index.html

```diff
@@ -55,7 +56,7 @@
         <div class="row">
           <div class="col-md-12 text-center">
             <div class="mdi-material-light-blue">
-              <h1>A GUI for BOSH</h1>
+              <h1>The GUI for BOSH</h1>
```

Comment:
Do we also want to use `The GUI for BOSH` on our T-Shirts instead of `GUI for BOSH`?